altair_pandas/test_plotting.py

```diff
@@ -63,3 +63,14 @@ def test_dataframe_hist(dataframe, with_plotting_backend):
     assert 'field' not in spec['encoding']['y']
     assert spec['encoding']['color']['field'] == 'column'
     assert spec['transform'][0]['fold'] == ['x', 'y']
+
+
+def test_boxplot(series, dataframe, with_plotting_backend):
+    for data in [series, dataframe]:
+        chart = data.plot.box()
+        spec = chart.to_dict()
+        assert spec['mark'] == 'boxplot'
+        assert spec['encoding']['x']['field'] == 'column'
+        assert spec['encoding']['y']['field'] == 'value'
+        fold = ['data_name'] if isinstance(data, pd.Series) else ['x', 'y']
+        assert spec['transform'][0]['fold'] == fold
```

Comment:
Can you split this into two functions? The reason is, if one starts to fail it's more clear exactly where the failure is. Here if series.plot.box() starts to fail, you have to add logging and re-run to figure out whether it was the series plot or the dataframe plot (or both) that failed.